### ISSUE TITLE
difficulties with installation 

### ISSUE THREAD

**Issue by gaelleclain** [CLIMADA-project/climada_python#834]
Hello, I have been following the installation guidelines both the simple instructions and the advanced instructions. Both have failed. I hope I can find help here.

I have a Windows computer and I work on a linux subsystem using wsl2.
The disk space and the internet connection requirements are adressed.

1. Installation topic #1
Today I have tried again to install climada using the advanced instructions together with this video [this video](https://youtu.be/P1Trj-BwRSY?si=pS-bggUdw3-aiusZ). I have installed anaconda and updated python to the latest version. I have been stuck on the environment creation step since more than 10 hours now. 
![image](https://github.com/CLIMADA-project/climada_python/assets/107561830/3ec7da4e-0447-42c5-9323-c5d100be5aa4)
 Yet is that a normal that the environment creation takes this long? 

2. Installation topic #2 using pycharm

Under pycharm I have created a new project.

In the pycharm terminal I have successfully: 
- updated pip 
- executed "pip install climada"
- 
Then I have switched to the Python console and executed "import climada" 
the installation seems ok but I have this issue:
if I ever execute "climada.entity" command I get :" ModuleNotFoundError: No module named 'osgeo'"
Can you point me what I have missed? 

Thank you

**Comment by peanutfun** [comment]
@gaelleclain Were you able to resolve the issue with the latest installation instructions? Note that the video you mention seems outdated regarding our recent changes to the installation procedure, and we are not affiliated with its creators in any way.

**Comment by gaelleclain** [comment]
@peanutfun thank you for your message. I have tested the installation procedure with Miniforge3 for Windows. 
Everything went fine but the test mentionned in the procedure:

![image](https://github.com/CLIMADA-project/climada_python/assets/107561830/1f0a1d71-0be4-4e04-9863-ef4b612a751e)

I seem to understand that Stamen is something related to geopandas that is outdated. Is there a solution for this ?

Thanks

**Comment by gaelleclain** [comment]
@peanutfun @chahank I have done [this ](https://github.com/CLIMADA-project/climada_python/pull/798#issuecomment-1893781486) modification and retested.
Same error message. 
Should I recompile or deactivate / reactivate climada_env? 
Please note that I have no clue how to recompile.

Thanks

**Comment by peanutfun** [comment]
@gaelleclain This is an issue with xyzservices. They removed the Stamen provider. We fixed this in the current development version of Climada. Since I think you installed Climada via Conda, I am not sure if you can fix this issue on your end. You might need to switch to the development version, following the "Advanced Instructions" in the installation guide.

A new Climada release including the fix will probably available by the end of the month from conda-forge.

**Comment by gaelleclain** [comment]
@peanutfun I have installed climada via Mamba. Does it make a difference on adressing the issue?
Otherwise I'll switch to the Advanced instruction installation indeed

**Comment by peanutfun** [comment]
@gaelleclain Yes. If you install Climada via Mamba, you only get the released versions. Currently, there is no release that fixes the issue you mentioned. Switching to the advanced instructions is probably your best course of action. It is a bit more complicated, but you do not require additional software.

**Comment by gaelleclain** [comment]
@peanutfun understood, thanks. 
so far I have done a dirty workaround like mentionned in [Goodbye Stamen](https://github.com/CLIMADA-project/climada_python/pull/798#top)
#798 
The test  "python -m unittest climada.engine.test.test_impact" has generated many warning messages but ended on a OK so I take it as a problem solved.
Thanks for helping indeed 
![image](https://github.com/CLIMADA-project/climada_python/assets/107561830/11b64f3e-b8ae-42ac-8e8f-abe49fbcfee0)

**Comment by peanutfun** [comment]
Resolved by the latest update to the Conda package, see https://github.com/conda-forge/climada-feedstock/pull/9.

To update, execute `mamba update -n climada_env climada`
